### ISSUE TITLE
crash on android with gyroscope

### ISSUE THREAD

**Issue by volzhs** [godotengine/godot#5779]
**Operating system or device - Godot version:**
Android 4.4 (Galaxy note 2), c328693e837f5899b24504caef5a9ee5e4d4cb4e 

**Issue description** (what happened, and what was expected):
Crash on android device because of gyroscope.

adb logs
```
FATAL EXCEPTION: main
Process: com.godot.game, PID: 29465
java.lang.UnsatisfiedLinkError: Native method not found: org.godotengine.godot.GodotLib.gyroscope:(FFF)V
	at org.godotengine.godot.GodotLib.gyroscope(Native Method)
	at org.godotengine.godot.Godot.onSensorChanged(Godot.java:678)
	at android.hardware.SystemSensorManager$SensorEventQueue.dispatchSensorEvent(SystemSensorManager.java:474)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:138)
	at android.os.Looper.loop(Looper.java:131)
	at android.app.ActivityThread.main(ActivityThread.java:5602)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
	at dalvik.system.NativeStart.main(Native Method)

```

I will try to make android template with current head and test it again.

**Comment by akien-mga** [comment]
Yes I recently merged #5718 so you probably need new templates.

**Comment by volzhs** [comment]
It works now. I got wrong android template with related #5645 issue.
